docs/throttling.md

```diff
@@ -130,3 +130,11 @@ Below is a table of the various `cpuSlowdownMultipliers` you might want to use t
 # Run Lighthouse with a custom multiplier
 lighthouse --throttling.cpuSlowdownMultiplier=6 https://example.com
 ```
+
+## Slow CPU Warnings in the Lighthouse Report
```

Comment:
is this what you were hoping for @paulirish ?

Reply:
i made edits. lots of 'em :)